src/Documentation/tutorials_and_samples/tutorial_1.md

```diff
@@ -198,10 +207,20 @@ namespace OrleansBasics
         {
             try
             {
-                using (var client = await ConnectClient())
+                var exitRequested = false;
+                while (!exitRequested)
                 {
-                    await DoClientWork(client);
-                    Console.ReadKey();
+                    using (var client = await ConnectClient())
+                    {
+                        var reconnectRequested = false;
+                        while (!reconnectRequested)
```

Comment:
This disconnect/reconnect functionality is rather unconventional. I would never expect that kind of behavior in a real-world application, or any behavior which involves disposing and recreating the client.

Reply:
agreed ... but then a sample application is not real-world either ... I just thought it made more fun than send one message and done

Reply:
I think this is likely to show an anti-pattern - of disposing a cluster client, even if that's not the intent of the sample. The simple and straightforward approach we recommend is to initialize a cluster client once, and then hold on to it for the life of the app.